### PR TITLE
[ROX-17772] Remove all dackbox stores

### PR DESCRIPTION
**Pull request by md2119**
## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### CODE REVIEW

File: central/deployment/datastoretest/datastore_sac_test.go [hunk 1]
```diff
@@ -48,11 +43,8 @@ type deploymentDatastoreSACSuite struct {
 	suite.Suite
 
 	// Elements for bleve+rocksdb mode
-	engine   *rocksdb.RocksDB
-	index    bleve.Index
-	dacky    *dackbox.DackBox
-	keyFence dackboxConcurrency.KeyFence
-	indexQ   queue.WaitableQueue
+	engine *rocksdb.RocksDB
```
Comment (connorgorman) [inline central/deployment/datastoretest/datastore_sac_test.go:46]: remove these?

File: central/image/datastoretest/datastore_sac_test.go [hunk 2]
```diff
@@ -41,11 +35,8 @@ type imageDatastoreSACSuite struct {
 	suite.Suite
 
 	// Elements for bleve+rocksdb mode
-	engine   *rocksdb.RocksDB
-	index    bleve.Index
-	dacky    *dackbox.DackBox
-	keyFence dackboxConcurrency.KeyFence
-	indexQ   queue.WaitableQueue
+	engine *rocksdb.RocksDB
```
Comment (connorgorman) [inline central/image/datastoretest/datastore_sac_test.go:38]: Remove?

**Review by connorgorman — APPROVED** [review]
Please address or resolve the comments if they don't make sense, then ship

File: central/cve/fetcher/manager_impl_postgres_test.go [hunk 3]
```diff
@@ -446,8 +435,7 @@ type TestClusterCVEOpsInPostgresTestSuite struct {
 
 	mockCtrl            *gomock.Controller
 	ctx                 context.Context
-	db                  postgres.DB
-	gormDB              *gorm.DB
+	testPostgres        pgtest.TestPostgres
```
[the remark this answers is not in the document]
Comment (md2119) [reply central/cve/fetcher/manager_impl_postgres_test.go:438]: not yet. Seems like the work was not complete. Anyways, there is a ticket for it.

File: central/deployment/datastoretest/datastore_sac_test.go (hunk 1 above)
Comment (md2119) [reply central/deployment/datastoretest/datastore_sac_test.go:46]: Have a follow-up to refactor `DackboxTestDataStore` interface. I will address your comment in it.

File: central/image/datastoretest/datastore_sac_test.go (hunk 2 above)
Comment (md2119) [reply central/image/datastoretest/datastore_sac_test.go:38]: removed

**Pull request merged**